apps/studio/components/interfaces/Organization/BillingSettings/PaymentMethods/CurrentPaymentMethod.tsx

```diff
@@ -66,11 +67,15 @@ const CurrentPaymentMethod = () => {
 
       <Button type="outline" asChild>
         {subscription?.payment_method_type === 'invoice' ? (
-          <Link
-            href={`/support/new?slug=${slug}&ref=no-project&message=${encodeURIComponent('I would like to change my payment method')}&category=${SupportCategories.BILLING}`}
-          >
-            Contact support
-          </Link>
+          <ContactSupportButton
+            category="BILLING"
+            subject="Unable to change payment method"
+            message="I'm unable to change my payment method"
```

Comment:
hmm the subject for this one should be "I would like to change my payment method"
since this isn't a bug - but because their payment type is through invoice

Reply:
in which case this isn't really a bug and shouldn't be logged into Sentry